evaluator/evaluate_baseline_test.go

```diff
@@ -0,0 +1,55 @@
+package evaluator
+
+import (
+	"app/base/core"
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"testing"
+
+	"github.com/RedHatInsights/patchman-clients/vmaas"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLimitVmaasToBaseline(t *testing.T) {
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+	configure()
+
+	originalVmaasData := getVMaaSUpdates(t)
+	expectedErrata := []string{"RH-1", "RH-2", "RH-100"}
+	assert.Equal(t, expectedErrata, errataInVmaasData(originalVmaasData))
+
+	// a system without baseline
+	system := models.SystemPlatform{ID: 5, RhAccountID: 1, BaselineID: nil}
+	vmaasData := getVMaaSUpdates(t)
+	err := limitVmaasToBaseline(database.Db, &system, &vmaasData)
+	assert.Nil(t, err)
+	assert.Equal(t, originalVmaasData, vmaasData)
+
+	// a system with baseline but nothing filtered out
+	system = models.SystemPlatform{ID: 3, RhAccountID: 1, BaselineID: utils.PtrInt(2)}
+	err = limitVmaasToBaseline(database.Db, &system, &vmaasData)
+	assert.Nil(t, err)
+	assert.Equal(t, expectedErrata, errataInVmaasData(originalVmaasData))
+
+	// a system with baseline and filtered errata
+	system = models.SystemPlatform{ID: 1, RhAccountID: 1, BaselineID: utils.PtrInt(1)}
+	vmaasData = getVMaaSUpdates(t)
+	expectedErrata = []string{"RH-100"}
+	err = limitVmaasToBaseline(database.Db, &system, &vmaasData)
+	assert.Nil(t, err)
+	assert.Equal(t, expectedErrata, errataInVmaasData(vmaasData))
```

Comment:
Nice test. I would just use direct values in asserts. It's better understandable to me, and shorter. But not a big deal.
~~~golang
assert.Equal(t, []string{"RH-100"}, errataInVmaasData(vmaasData))
~~~

Reply:
fixed tests